package.json

```diff
@@ -37,7 +37,7 @@
 		"react-dnd-html5-backend": "^14.0.0"
 	},
 	"devDependencies": {
-		"react": "^17.0.2",
+		"react": "link:../protocol-design-fe/node_modules/react",
```

Comment:
This makes it super dependent on environment.  I'll capture as backlog ticket (i.e. looking at ways we can package this a bit better)